File: CS-Examples/10_Conversion/AddCoverImage.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Doc;
using Spire.Doc.Fields;
namespace AddCoverImage
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            // Create a new instance of the Document class.
			Document doc = new Document();

			// Load a document from the specified file path.
			doc.LoadFromFile(@"..\..\..\..\..\..\..\Data\ToEpub.doc");

			// Create a new DocPicture object with the document as its owner.
			DocPicture picture = new DocPicture(doc);

			// Load an image from the specified file path and assign it to the picture.
			picture.LoadImage(Image.FromFile(@"..\..\..\..\..\..\..\Data\Cover.png"));

			// Specify the output file name for the EPUB file.
			string result = "AddCoverImage.epub";

			// Save the document as an EPUB file, including the cover image.
			doc.SaveToEpub(result, picture);

			// Dispose the document object to release resources.
			doc.Dispose();
            WordDocViewer(result);
        }
        private void WordDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB-Examples/10_Conversion/AddCoverImage.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Doc
Imports Spire.Doc.Fields
Namespace AddCoverImage
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Create a new Document object
			Dim doc As New Document()

			' Load the document from the specified file path
			doc.LoadFromFile("..\..\..\..\..\..\..\Data\ToEpub.doc")

			' Create a new DocPicture object using the document
			Dim picture As New DocPicture(doc)

			' Load the image from the specified file path and assign it to the DocPicture
			picture.LoadImage(Image.FromFile("..\..\..\..\..\..\..\Data\Cover.png"))
			' =============================================================================
			' Use the following code for netstandard dlls
			' =============================================================================
			'picture.LoadImage("..\..\..\..\..\..\..\Data\Cover.png")
			' =============================================================================

			' Define the output file name for the EPUB file with the cover image added
			Dim result As String = "AddCoverImage.epub"

			' Save the document to an EPUB file, including the cover image, with the specified output file name
			doc.SaveToEpub(result, picture)

			' Dispose of the Document object to release resources
			doc.Dispose()

			WordDocViewer(result)
		End Sub
		Private Sub WordDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace
